src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -128,34 +133,52 @@ class PhoneEditModal extends React.Component {
     </div>
   );
 
+  renderForm = (formButtons, onSubmit) => {
+    if (!useNewForm) {
+      return this.renderOldForm();
+    }
+    return (
+      <ContactInfoForm
+        formData={this.props.field.value}
+        formSchema={this.props.field.formSchema}
+        uiSchema={this.props.field.uiSchema}
+        onUpdateFormData={this.props.onChangeFormDataAndSchemas}
+        onSubmit={onSubmit}
+      >
+        {formButtons}
+      </ContactInfoForm>
+    );
+  };
+
   render() {
     return (
       <Vet360EditModal
         getInitialFormValues={this.getInitialFormValues}
         render={this.renderForm}
-        onBlur={this.onBlur}
+        onBlur={useNewForm ? null : this.onBlur}
+        useSchemaForm={useNewForm}
         {...this.props}
       />
     );
   }
 }
 
 export function mapStateToProps(state, ownProps) {
-  const { fieldName } = ownProps;
```

Comment:
This was a little weird... must have been leftover from a previous refactor or something.